File: backend-java/sahayaSetu/src/main/java/com/sahayaSetu/dtos/ResourceRequestResponseDto.java
package com.sahayaSetu.dtos;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class ResourceRequestResponseDto extends RequestResponseDto {
    private String resourceType;
    private int quantityRequired;
    private int quantityReceived;
}

File: backend-dotnet/SahayaSetu/DTO/ResourceRequestResponseDto.cs
using SahayaSetu.Models;

namespace SahayaSetu.DTO
{
    public class ResourceRequestResponseDto : RequestResponseDto
    {
        public string? ResourceType { get; set; }
        public int QuantityRequired { get; set; }
        public int QuantityReceived { get; set; }
    }
}
